radicle-cli/src/commands/patch/common.rs

```diff
@@ -51,6 +52,7 @@ pub fn get_merge_target(
         anyhow::bail!("commits are already included in the target branch; nothing to do");
     }
 
+    crate::git::anonymous_fetch(workdir, &git::Url::from(storage.id), &target_oid)?;
```

Comment:
It's probably not a big deal, but is it necessary to always fetch?

Reply:
True